rewrite-xml/src/main/antlr/XMLLexer.g4

```diff
@@ -65,7 +65,7 @@ DTD_CLOSE        :  '>'    -> popMode ;
 DTD_SUBSET_OPEN  :  '['    -> pushMode(INSIDE_DTD_SUBSET) ;
 DTD_S            :   S     -> skip ;
 
-DOCTYPE          :  'DOCTYPE' ;
+DOCTYPE options { caseInsensitive = true; }         :  'DOCTYPE' ;
```

Comment:
We do have the option to apply this to the entire file instead of specifically `DOCTYPE`